aptos-move/framework/aptos-framework/sources/transaction_validation.spec.move

```diff
@@ -175,6 +175,7 @@ spec aptos_framework::transaction_validation {
 
         aborts_if !account::exists_at(gas_payer);
         aborts_if !(fee_payer_public_key_hash == account::get_authentication_key(gas_payer));
+        aborts_if !features::spec_fee_payer_enabled();
```

Comment:
how come the prover accepted this before without this

Reply:
Because there is a pragma above set to 120s which means only when the global time out is set to greater or equal to 120s, this function will be verified. Currently the timeout is set to 80s by default and the verification is skipped